lib/src/connection.dart

```diff
@@ -79,6 +79,7 @@ class PostgreSQLConnection implements PostgreSQLExecutionContext {
     _connectionState.connection = this;
   }
 
+  final StreamController<Notification> _notifications = new StreamController<Notification>.broadcast();
```

Comment:
Close this resource in `PostgreSQLConnection.close()`; make sure the future returned by `PostgreSQLConnection.close()` doesn't complete until `StreamController.close()`'s future completes